src/pages/UpdateServers.js

```diff
@@ -124,15 +124,16 @@ class UpdateServers extends BaseUpdateWizardPage {
           };
         });
       }
+      let model;
```

Comment:
There is still a problem here: if this.state.internalModel is already populated, then model will remain undefined, and line 136 will be passing undefined. I think the line above probably needs to be `let model = this.state.internalModel;`

Reply:
Good catch, we can use that as the default value for the model to address this.